### ISSUE TITLE
Tiny draggable surface on Win10

### ISSUE THREAD

**Issue by SergheiGurgruov** [minbrowser/min#836]
As you can see in the **screenshot below** , in on Windows10 there is a problem with the draggable surface of the window which is **way too small**.

_"it's really easy to just click on the tab below and it is quite frustrating.
expecially when you work with multiple monitors and need to move your windows very often."_

![image](https://user-images.githubusercontent.com/48824703/66823308-41ab3800-ef46-11e9-94a5-b76029e2fd76.png)

**Comment by PalmerAL** [comment]
I agree it is pretty small, but I'm not sure there's a perfect solution for this. Due to the way Windows works, any given part of the window can be draggable or clickable, but not both, so you can't drag the tabs to move the window - there has to be a separate area for it. Making the region at the top bigger makes dragging easier, but also makes it harder to click on tabs; one alternative solution would be to add a drag area in the upper-left corner, but that makes it harder to reach the menu button.

There's been a bunch of previous discussion about this here: #630

Now that the drag area is hidden while the window is maximized, we could probably make it a few pixels taller, but I'm not sure how much that would actually help; if you have any other ideas on this I'd be interested to hear them.
